src/NUnitFramework/FrameworkVersion.cs

```diff
@@ -26,5 +26,5 @@
 //
 // Current version for the NUnit Framework
 //
-[assembly: AssemblyVersion("3.14.0.0")]
-[assembly: AssemblyFileVersion("3.14.0.0")]
```

Comment:
It sucks that we have some 3.14 versions on AppVeyor, but I'm no longer planning on doing a 3.14 release.